lib/watcher.js

```diff
@@ -59,6 +59,12 @@ module.exports = class Watcher extends EventEmitter {
       return;
     }
     logger.debug('change');
+
+    // Communicate with the watchedRoot what file has changed.
+    // TODO: Is this the correct place to increment this or should be before 
+    // the above checks or even below as apart of the promise chain.
+    watchedRoot._revision++;
```

Comment:
rather then mutating internal state, could we make a public method `watchedRoot.revised()` or something?

Reply:
This may also be in the wrong place, as it is guarded by `this._ready`, but changes prior to ready, although strange, should likely still trigger a revision.

Thinking about this, we could likely just more `watchedRoot.revised()` into line 13 of the watch-adapter (the watcher.on event handler).

Reply:
Done in latest commit